roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -146,7 +146,7 @@ openstack_lbaas_enabled: false
 ## the k8s cluster. Only 'AlwaysAllow','AlwaysDeny', and
 ## 'RBAC' modes are tested.
 authorization_modes: []
-rbac_enabled: "{{ 'RBAC' in authorization_modes }}"
+rbac_enabled: "{{ 'RBAC' in authorization_modes or kubeadm_enabled }}"
```

Comment:
I think `authorization_modes: ['RBAC']`  needed for provide [correct apiserver flags](https://github.com/kubernetes-incubator/kubespray/blob/master/roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2#L87-L89)

Reply:
For kubeadm it's automatically enabled. We don't have to add it manually to the config file.

Reply:
and [this](https://github.com/kubernetes-incubator/kubespray/blob/bd272e0b3cdaecd9fb87b010a10f7a03d2721028/roles/kubernetes/master/templates/kubeadm-config.yaml.j2#L22-L24)?)

Reply:
Think of this field as "extra authorization modes". RBAC and Node actually both get added by default